typings/artifacts.d.ts

```diff
@@ -314,6 +314,7 @@ declare global {
 
       export interface LanternMetric {
         timing: number;
+        timestamp?: never;
```

Comment:
allows you to check existence of timestamp to see if it's `Metric` or `LanternMetric`